deepmd_utils/model_format/network.py

```diff
@@ -409,3 +411,98 @@ def deserialize(cls, data: dict) -> "EmbeddingNet":
         obj = cls(**data)
         super(EmbeddingNet, obj).__init__(layers)
         return obj
+
+
+class Networks:
```

Comment:
perhaps the name `NetworkCollection` or `NetworkArray` is more self-explanatory.